### ISSUE TITLE
PIP with back/home broken on shield

### ISSUE THREAD

**Issue by jeyjai** [yuliskov/SmartTube#1248]
After Nvidia shield updated to Android 11 i started having problems with videos crashing but only if PIP is on and set to home/back.
Basically every 2-3 videos crash about 1-3 seconds in.
It happens to all videos randomly, they crash more depending on order played.
Its never the first video crashing, always second or third.
This happens in both the latest stable and beta versions as soon as i change those two settings.

**Comment by Harmonie666** [comment]
I have the same. I own Shield 2019 pro. Is this an Nvidia shield issue? It happens pretty often that when I open a video it closes(crashes) right away:

https://user-images.githubusercontent.com/73917917/150827990-5f14c316-6b5b-497a-bdba-a765e722ca33.mp4

**Comment by JimZiii** [comment]
Yes, it seems to happen every other video. So the first video i start works, the second crash and i have to restart it, then you're in this loop of restarting every video so the next video crash and also needs to be restarted.

Another issue i noticed at the same time is the play/pause button on the shield no longer works as it used to. If i press play/pause in pip the button instead works as the ok button. If i got the same video highlighted it just fullscreens it and if i press pause with another video highlighted it starts that video instead. But with a video in fullscreen play/pause works as intended. I remember being able to play/pause the pip video before all this happened.

**Comment by JimZiii** [comment]
Guess

**Comment by thefloppy** [comment]
I have the same Problem with my shield.
I can not play videos one after another since the last experince 9 Update.
